.circleci/config.yml

```diff
@@ -689,6 +714,7 @@ workflows:
           name: codedeploy_fargate_tear-down-test-env
           requires:
             - codedeploy_fargate_test-update-service-job
+            - codedeploy_fargate_test-update-and-wait-service-job
```

Comment:
We can remove `codedeploy_fargate_test-update-service-job` if the 2 jobs will be run in sequence